app/routes/user.email._index/EmailNotificationCard.tsx

```diff
@@ -151,6 +154,51 @@ export default function EmailNotificationCard({
           </ModalToggleButton>
         </ModalFooter>
       </Modal>
+      <Modal
+        id="test-confirmation"
+        ref={testConfirmRef}
+        aria-labelledby="modal-confirm-heading"
+        aria-describedby="modal-confirm-description"
+        renderToPortal={false} // FIXME: https://github.com/trussworks/react-uswds/pull/1890#issuecomment-1023730448
+      >
+        <ModalHeading id="modal-confirm-heading">
+          Test Email Confirmation
+        </ModalHeading>
+        <p id="modal-confirm-description">
+          This will send a generic test email to {recipient}. Do you wish to
+          continue?
+        </p>
+        <testFetcher.Form method="POST">
+          <ReCAPTCHA
+            onChange={(value) => {
+              setRecaptchaValid(Boolean(value))
+            }}
+          />
+          <ModalFooter>
+            <input type="hidden" name="recipient" value={recipient} />
+            <input type="hidden" name="intent" value="sendTest" />
+            <ModalToggleButton
+              data-close-modal
+              modalRef={testConfirmRef}
+              closer
+            >
+              Cancel
+            </ModalToggleButton>
+            <Button
+              type="submit"
+              outline
+              data-close-modal
+              disabled={!(recaptchaValid && testFetcher.state === 'idle')}
+            >
+              <Icon.MailOutline
+                role="presentation"
+                className="bottom-aligned margin-right-05"
+              />
+              Send
+            </Button>
```

Comment:
Put these in a button group and fix the alignment:

![Screenshot 2024-09-03 at 19 45 46](https://github.com/user-attachments/assets/b6406a17-9474-4d76-a0cc-26c3ccfaddf8)